doc/usage/extensions/autosummary.rst

```diff
@@ -74,13 +74,22 @@ The :mod:`sphinx.ext.autosummary` extension does this in two parts:
 
      The ``toctree`` option also signals to the :program:`sphinx-autogen` script
      that stub pages should be generated for the entries listed in this
-     directive.  The option accepts a directory name as an argument;
+     directive.
+
+     The option accepts a directory name as an argument;
      :program:`sphinx-autogen` will by default place its output in this
      directory. If no argument is given, output is placed in the same directory
-     as the file that contains the directive.
+     as the file that contains the directive.  The directory is interpreted as
+     relative to the directory of the source file, unless it begins with the
+     system path separator (``/`` for Unix-likes), in which case it is relative
```

Comment:
Does it mean that for Windows, it should be `\` ? You shouldn't say "system path separator" otherwise. Also you are using `posixpath` so I am assuming that you only want to treat `/`.

Reply:
Oh thanks for the catch - it's out-of-date documentation.  Originally I had written it to use `os.sep`, but then I realised that the whole directory handling in `autosummary` at the moment requires Unix-like paths and that it'd just make everything inconsistent for Windows builds to use different relative separators.

I've updated this documentation in 62b27c495.